File: baekjoon/02_Conditional_Statements_if/9498.py
score = input()
score = int(score)

if 90 <= score and score <= 100:
    print("A")
elif 80 <= score and score <= 89:
    print("B")
elif 70 <= score and score <= 79:
    print("C")
elif 60 <= score and score <= 69:
    print("D")
else:
    print("F")

File: C/2022-12-11/9498.c
#include <stdio.h>

int main() {
    int score;
    
    scanf("%d", &score);
    if(90 <= score && score <= 100) printf("A\n");
    else if (80 <= score && score <= 89) printf("B\n");
    else if (70 <= score && score <= 79) printf("C\n");
    else if (60 <= score && score <= 69) printf("D\n");
    else printf("F\n");
    return 0;
}
